docs/openapi/schemas/StatusListCredential.yml

```diff
@@ -0,0 +1,41 @@
+title: Status List Verifiable Credential
+type: object
+allOf:
+  - $ref: './Credential.yml'
+  - type: object
+    properties:
+      proof:
+        $ref: './CredentialLinkedDataProof.yml'
+required:
+  - id
+example:
+  {
+    '@context': [
+      'https://www.w3.org/2018/credentials/v1',
+      'https://w3id.org/vc/status-list/2021/v1'
```

Comment:
100% needed for v1 ... 100% an error for v2

Reply:
Just updated this to use the mesur prod api